package.json

```diff
@@ -4,8 +4,8 @@
   "dependencies": {
     "@hotwired/stimulus": "^3.2.1",
     "@opensystemslab/map": "^0.5.7",
-    "@rails/activestorage": "^6.0.0",
-    "@rails/ujs": "^6.0.0",
+    "@rails/activestorage": "^7.0.4-2",
+    "@rails/ujs": "^7.0.4-2",
```

Comment:
Do we need `@rails/ujs` at all? It seems like it's the predecessor to turbo, which we also don't use. (I have no idea about `@rails/activestorage`.)